packages/pillow/meta.yaml

```
@@ -12,8 +12,8 @@ source:
       - src/setup.cfg
       - ./setup.cfg
 build:
-  cflags: -s USE_ZLIB=1 -s USE_LIBJPEG=1 -s USE_FREETYPE=1
-  ldflags: -L ../../../../emsdk/emsdk/fastcomp/emscripten/cache/asmjs/ ../../../../emsdk/emsdk/fastcomp/emscripten/cache/asmjs/libjpeg.bc
+  cflags: -s USE_ZLIB=1 -s USE_LIBJPEG=1 -s USE_FREETYPE=1 -fPIC
```

This is already in the default fclags
```suggestion
  cflags: -s USE_ZLIB=1 -s USE_LIBJPEG=1 -s USE_FREETYPE=1
```